<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyramidDialog</class>
 <widget class="QDialog" name="PyramidDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pyramidal decomposition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Type of pyramid</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="gaussianButton">
        <property name="text">
         <string>gaussian</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="laplacianButton">
        <property name="text">
         <string>laplacian</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="filterBox">
       <item>
        <property name="text">
         <string>triangular</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gaussian</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>trimodal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rectangular</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>qmf</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nbStepLabel">
       <property name="text">
        <string>Number of steps : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="stepBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="oneStepBox">
       <property name="text">
        <string>Create only one step :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="onlyStepBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PyramidDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PyramidDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oneStepBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>onlyStepBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oneStepBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>nbStepLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oneStepBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>stepBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
